utils/validateSettingsFile.js

```diff
@@ -49,4 +50,4 @@ const validateSettingsFile = (settingsFilePath) => {
   return true;
 };
 
-module.exports = { validateSettingsFile };
+export { validateSettingsFile };
```

Comment:
If not default needed, maybe would ne better to put export in the declaration above, for example 

`export const validateSettingsFile = () => {`

Reply:
Will do it. Thanks~!